server/knexfile.js

```diff
@@ -0,0 +1,50 @@
+// Update with your config settings.
+
+/**
+ * @type { Object.<string, import("knex").Knex.Config> }
```

Comment:
What is this @type ?

Reply:
Good question, I'm not sure, that was part of the code that was auto-generated when we made this file